src/client/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "precommit": "lint-staged",
-    "start": "cross-env REACT_APP_BUILD_VERSION=$(git describe) react-scripts start",
+    "start": "react-scripts start",
```

Comment:
Q: How come this was changed? Looks unrelated to the issue at hand.

Reply:
We changed it to get `npm start` working on @zhouyuhe 's machine - I added that bit, but it only seems to work on Mac/Unix 😞

Reply:
Because with `REACT_APP_BUILD_VERSION=$(git describe)` it shows the error, and I cannot run the app